Clarify Safari's support of MathML

Xyene opened the pull request:
Refs #2734; Safari 10+ fixes the issues referenced in the latter ticket.

The [Mozilla MathML torture test](https://developer.mozilla.org/en-US/docs/Mozilla/MathML_Project/MathML_Torture_Test) doesn't have any issues in rendering common math symbols (e.g. summations, which were fairly buggy before). RTL support is pretty good (radicals render properly), and outstanding rendering issues with RTL math seem to be buggy rather than downright unsupported.

Fyrd commented:
I suspect you meant this to change to `a` for partial support? `p` refers to "supported with polyfill".

Additionally you should add a numbered note to explain the partial support. Thanks!

Xyene commented:
My bad, I made a mental note not to get tripped up by it, but still did.

I've added number notes to the Safari data, and also added a direct link to the MathML torture test it references.

Fyrd commented:
Looks great, thanks!
